package.json

```diff
@@ -19,16 +19,16 @@
     "grunt": "~0.4.5",
     "grunt-cli": "0.1.13",
     "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-coffee": "~0.11.0",
+    "grunt-contrib-coffee": "~0.13.0",
```

Comment:
I would assume we want the dependencies on the repo to be consistent with the package.json, right?